src/app/clusters/time-synchronization-server/time-synchronization-server.cpp

```diff
@@ -37,10 +34,12 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/RuntimeOptionsProvider.h>
 
-#include <app-common/zap-generated/cluster-enums.h>
-
 #include <system/SystemClock.h>
 
+#if TIME_SYNC_ENABLE_TSC_FEATURE
```

Comment:
Is this including the relevant config header?  I don't see it.

Reply:
the define is used in time-synchronization-server which includes the right include. You want to include it again?

Reply:
If time-synchronization-server.h is including it, probably ok.